packtools/sps/formats/sps_xml/kwd_group.py

```diff
@@ -0,0 +1,35 @@
+"""
+data = {
+    "kwd_lang": "pt",
+    "kwd_title": "Palavra-chave",
+    "kwds": [
+        "Broncoscopia",
+    ]
+}
+"""
+
+import xml.etree.ElementTree as ET
+
+
+def build_kwd_group(data):
+    kwd_lang_text = data.get("kwd_lang")
+    if not kwd_lang_text:
+        raise ValueError("kwd lang is required")
```

Comment:
@Rossi-Luciano Use:
```python
try:
     kwd_lang_text = data["kwd_lang"]
except KeyError:
    raise ValueError("kwd_lang is required")
```
Note que você tem que informar o nome da chave que é exigida

Reply:
@robertatakenaka nesse caso quando o valor for `None` a exceção não ocorrerá.